Release 3.18.0 (#1046)
GeotrekCE/Geotrek-rando-v3: docs/changelog.md

# Changelog

## 3.18.0 (2024-02-07)

**🚀 New features**

> - Add HD viewpoints to treks/outdoorCours/POIs (#943)

Reviewer: ```suggestion
- Add HD viewpoints on treks, outdoor courses and POIs (#943)
```
Les vues HD ne sont pas aussi sur les sites outdoor ?

Author: Fixed

> 

Reviewer: Ajouter : 
```
**🐛 Fixes**

- Fix header customization regression in version 3.17.0 (#1019)
- Fix scriptsHeader/Footer containing only one object, regression since 3.15.4 (#1029)
```

Dans maintenance, ajouter : 
```
- Update Cypress and development documentation (#1028)
```

Author: Ces fonctionnalités ont été ajouté lors de la release précédente https://github.com/GeotrekCE/Geotrek-rando-v3/releases/tag/v3.17.1

Reviewer: Ah oui je me suis embrouillé !
